apple/testing/default_runner/ios_sample_test_runner.template.sh

```diff
@@ -30,13 +30,6 @@ basename_without_extension() {
 
 # Unpack the output IPA into a tmp folder
```

Comment:
perhaps reword this comment to "Create tmp folder to contain the extracted .xctest bundle", and then add a comment where we extract the test bundle in line 52